rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/policy/RMAwareNodeSourcePolicy.java

```diff
@@ -76,8 +76,8 @@ public void shutdown(Client initiator) {
         if (!rmShuttingDown) {
             try {
                 rmMonitoring.removeRMEventListener();
-            } catch (RMException e) {
-                logger.error("Error when removing event listener", e);
+            } catch (Exception e) {
```

Comment:
Can we be a bit more selective on the exception to be intercepted ?

Reply:
same answer